src/interfaces/chain.h

```diff
@@ -269,6 +269,9 @@ class Chain
     //! Register handler for RPC. Command is not copied, so reference
     //! needs to remain valid until Handler is disconnected.
     virtual std::unique_ptr<Handler> handleRpc(const CRPCCommand& command) = 0;
+
+    //! Notify handler of all memory pool transactions.
+    virtual void notifyMemoryPoolTransactions(Notifications& notifications) = 0;
```

Comment:
Could you call this notifyMempoolTransactions? I'd like to converge on _mempool_ spelling rather than _memPool_ or _memoryPool_. Currently we have:

<pre>
Chain::hasDescendantsIn<i>Mempool</i>
Chain::<i>mempool</i>MinFee
Chain::Notifications::TransactionAddedTo<i>Mempool</i>
Chain::Notifications::TransactionRemovedFrom<i>Mempool</i>
Chain::Lock::submitTo<i>MemoryPool</i>
</pre>

so the only existing exception is the `Chain::Lock` method, which I want to go away anyway.

Reply:
Will do.